### PR TITLE
Copyright

### PR DESCRIPTION
**Pull request by moxiegirl**
Updating copyright material for Documentation.
Also, I'm running this by Legal...we have a _Legal_ section, a NOTICE, and two copyrights now.

### CODE REVIEW

**Comment by LK4D4** [comment]
@moxiegirl You have commit "Remove bad file" which doesn't seem to make any changes in combine with next commit.

**Comment by moxiegirl** [comment]
@LK4D4 Thanks for the catch. I've rebased and sorted this.

File: README.md [hunk 1]
```diff
@@ -271,9 +271,12 @@ For more information, please see https://www.bis.doc.gov
...
 =========
-Docker is licensed under the Apache License, Version 2.0. See
-[LICENSE](https://github.com/docker/docker/blob/master/LICENSE) for the full
-license text.
+Copyright © 2014-2015 Docker, Inc. All rights reserved, except as follows:
+
+* Except where otherwise noted, the code in this repository is licensed under the Apache License, Version 2.0. See [LICENSE](https://github.com/docker/docker/blob/master/LICENSE) for the full license text. 
+
+* The files in the `docs` folder are licensed to end users under the Creative Commons Attribution 4.0 International License under the terms and conditions set forth in the file [`LICENSE-DOCS`](LICENSE-DOCS). You may
+obtain a duplicate copy of the same license, titled CC-BY-SA-4.0, at http://creativecommons.org/licenses/by/4.0/.
```
Comment (SvenDowideit) [inline README.md]: What's the license on the man page dir?

I suspect we'd be better off licensing all markdown / docs files as CC or Apache - as stated, there is no license stated for

Comment (thaJeztah) [inline README.md]: Maybe "the code in this repository" (which would need a definition of "code"), should be replaced with "the files in this repository"

**Comment by moxiegirl** [comment]
@SvenDowideit @thaJeztah Thanks guys, I've sent an email to Legal to review.  I'll wait to hear from them.

**Comment by thaJeztah** [comment]
Thanks @moxiegirl. Sorry for the bike-shedding (again) :relaxed:

**Comment by thaJeztah** [comment]
ping @moxiegirl did you get a response on this one as well?

**Comment by moxiegirl** [comment]
@thaJeztah not yet. :-(

**Comment by thaJeztah** [comment]
So at which desk should I put my stroopwafels to get this moved forward? :smile:

**Comment by moxiegirl** [comment]
Well, we won't solve it before 1.9.  The Legal Team is not local...they are consultants which is most likely why they are immune to waffels.  I'll start kicking again after the release goes out....or you can try attacking it from the Opens Source end...I'll send you an email.

**Comment by thaJeztah** [comment]
Thanks!

**Comment by LK4D4** [comment]
ping @thaJeztah @moxiegirl

**Comment by thaJeztah** [comment]
@LK4D4 I've also e-mailed the legal team, but unfortunately got no response, so we cannot merge this yet :cry:

**Comment by LK4D4** [comment]
@thaJeztah I propose to close it now, then. We have exceptionally high amount of PRs now :(

**Comment by thaJeztah** [comment]
Agree, I'll close until we get a reply, because we won't be able to move forward until that

**Pull request closed without merging**